package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "kytos-web-ui",
   "description": "Kytos-NG Web-ui project",
-  "version": "2022.3.0",
+  "version": "2023.1.0.b1",
```

Comment:
Format must be like "2023.1.0-b1" to execute the npm run build

Reply:
Right. Fixed here https://github.com/kytos-ng/ui/pull/44/commits/00486fff029b3ee2a509f93f79c90da171731130